messages/eslintEngine.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
-	"InvalidPath": "Invalid path in 'overriddenConfigPath' property of Config.json: %s",
-	"ConfigFileMissing": "Cannot find %s with overridden path %s"
-};
\ No newline at end of file
+	"InvalidPath": "The path in the `overriddenConfigPath` property of Config.json is invalid. %s.",
```

Comment:
Can we print the actual full file path to Config.json here as well?

Reply:
Probably.

Reply:
Actually, it seems that this message is never used. So I'm not going to worry about it.